automatic/brave/tools/chocolateyInstall.ps1

```diff
@@ -3,8 +3,12 @@
 
 $packageArgs = @{
   packageName = $env:ChocolateyPackageName
-  file        = "$toolsPath\BraveBrowserStandaloneSilentSetup32.exe"
-  file64      = "$toolsPath\BraveBrowserStandaloneSilentSetup.exe"
+  url                    = ''
+  url64bit               = ''
+  checksum               = ''
+  checksum64             = ''
+  checksumType           = 'sha256'
+  checksumType64         = 'sha256'
```

Comment:
Not sure if I agree with moving this to a completely remote package (a package that only downloads during installation).

If the 64bit installer isn't too large, the 64bit installer should still be embedded but download the 32bit if requested.

Can you please make the necessary changes to make that happen?